assets/delivery-options/js/myparcel.js

```diff
@@ -19,31 +19,35 @@ MyParcel = {
         this.data = myParcelConfig;
 
         isMobile     = true;
-        if(jQuery( window ).width() > 980 ) {
+        if(jQuery(window).width() > 980) {
             isMobile = false;
         }
 
-        /* Titels of the options*/
-        if (MyParcel.data.config.deliveryTitel){
-            jQuery('#mypa-delivery-titel').html(MyParcel.data.config.deliveryTitel);
+        /* Title's of the options*/
```

Comment:
In het engels zijn `'s` gelijk aan `is`
dit moet titles zijn:
https://translate.google.nl/?hl=nl#nl/en/titels

Reply:
Done!